doc/links.txt

```diff
@@ -22,7 +22,8 @@
 
 .. _`Socket API`: https://pubs.opengroup.org/onlinepubs/9699919799/functions/socket.html
 
-.. _`ZBOSS API documentation`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/index.html
+.. _`ZBOSS API documentation`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss_api_doc/index.html
+.. _`ZBOSS stack v3.3.0.4 release notes`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/3.3.0.4/zboss_release_notes.html
```

Comment:
If this PR gets merged: https://github.com/nrfconnect/sdk-nrf/pull/3205
Please update the link name /or/ move the doc upload and change both: the link name and the URL.

Reply:
@b-gent , can you help with this?

Reply:
@b-gent , can you help with updating the directory name nad the landing page?

Reply:
Do we really need to mention the version in this link? The idea of having a separate HTML page was to maintain all version information there, without worrying about it in NCS links.